### PR TITLE
Added two new small areas to maintenence

### PR DESCRIPTION
**Pull request by SethLafuente**
## About the PR 
Added two new small areas to maintenance personally I think maintenance needs something more other than just hallways and I guess this is kind of two small maintenance areas to a small space station 
**Screenshots**
image
image

**Changelog**

:cl: Seth
- add: Added two new maintenance areas

### CODE REVIEW

File: Resources/Maps/saltern.yml [hunk 1]
```diff
@@ -12336,6 +12336,46 @@ entities:
...
+      - 0
+      moles:
+      - 16.36866
+      - 61.57734
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
+      - 0
```
Comment (Zumorica) [inline Resources/Maps/saltern.yml:12378]: Looks like you fucked up the atmosphere on the grid.

**Comment by SweptWasTaken** [comment]
https://hackmd.io/@ss14/mapping-guide

How to fix roundstart atmos after making changes to the map

    fixgridatmos 1 Where “1” is the grid ID. This clears out all the air.
    You’re done! You can now save the map.

File: Resources/Maps/saltern.yml (hunk 1 above)
Comment (SethLafuente) [reply Resources/Maps/saltern.yml:12378]: a alright I can fix that

Comment (SethLafuente) [reply Resources/Maps/saltern.yml]: its fixed right test it and the atmos is fine

**Review by SweptWasTaken — APPROVED** [review]

**Comment by SweptWasTaken** [comment]
Pretty sure this is g2g.

**Comment by TimrodDX** [comment]
If I may, I think firelocks should be added to those new doors, if only because it doesn't really make a lot of sense (to me at least) as to why there'd be a bypass for the existing firelocks placed directly next to them. Then again, I might do that myself if you just want to merge it, since I feel like these rooms need more than just a table and a locker to justify their existence.

Personally, if I were doing this, I would make them sort of like the break rooms in the sewer areas in the Final Fantasy 7 remake, but that's just me.

**Comment by SethLafuente** [comment]
> If I may, I think firelocks should be added to those new doors, if only because it doesn't really make a lot of sense (to me at least) as to why there'd be a bypass for the existing firelocks placed directly next to them. Then again, I might do that myself if you just want to merge it, since I feel like these rooms need more than just a table and a locker to justify their existence.
> 
> Personally, if I were doing this, I would make them sort of like the break rooms in the sewer areas in the Final Fantasy 7 remake, but that's just me.

Sure I can add firelocks into the doors when it comes to it having more stuff part of the purpose of this at least why I made it just to add some extra rooms antags can do secret stuff in an such it isn't a special place just maintenance You got any recommendations on what can be on those rooms the lockers could have a small chance of dropping each type tool like a 0.1 prob chance per tool

**Comment by TimrodDX** [comment]
image

image


Here's a couple of examples of what I mean, based off the rooms you made there. The first one is now an electrical storage closet that can easily be used to hide a body, or something else, while also looking like it has a purpose. The second one is a storage area for cargo with a steel sheet crate in it (the other one is empty). I think this gives these small storage areas a bit more character.

**Comment by SethLafuente** [comment]
> image
> 
> image
> 
> Here's a couple of examples of what I mean, based off the rooms you made there. The first one is now an electrical storage closet that can easily be used to hide a body, or something else, while also looking like it has a purpose. The second one is a storage area for cargo with a steel sheet crate in it (the other one is empty). I think this gives these small storage areas a bit more character.

If you want to add that over mine on the same location I don't really mind but I don't feel like remaking it

**Pull request merged**